app/models/folio/checkout.rb

```diff
@@ -217,16 +217,16 @@ def renewal_count
       record.dig('details', 'renewalCount') || 0
     end
 
-    # nil means "unlimited" for unseenRenewalsRemaining
     def unseen_renewals_remaining
-      Float::INFINITY
-      # fields['unseenRenewalsRemaining'] || Float::INFINITY
+      unseen_renewals_allowed - renewal_count
+    end
+
+    def unseen_renewals_allowed
+      record.dig('details', 'loanPolicy', 'renewalsPolicy', 'numberAllowed') || Float::INFINITY
```

Comment:
If there isn't a renewals policy, should we default to 0?

```suggestion
      record.dig('details', 'loanPolicy', 'renewalsPolicy', 'numberAllowed') || 0
```

Reply:
I guess? I wasn't sure if the nil = unlimited still applied in Folio land: https://github.com/sul-dlss/mylibrary/blob/main/app/models/symphony/checkout.rb#L210

Reply:
🤔